docs/kpod-stop.1.md

```diff
@@ -0,0 +1,35 @@
+% kpod(1) kpod-stop - Stop one or more containers
+% Brent Baude
+# kpod-stop "1" "September 2017" "kpod"
+
+## NAME
+kpod stop - Stop one or more containers
+
+## SYNOPSIS
+**kpod** **stop** [*options* [...]] container
+
+## DESCRIPTION
+Kpod stop will stop one or more containers.  You may use container IDs or names as input.  The **--timeout** switch
```

Comment:
nit: bold `kpod stop` just to keep it same as the other docs

Reply:
fixed